src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,120 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.lib.DataProcessingException;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+public class ManufacturerDaoImpl implements ManufacturerDao {
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String manufacturerName = manufacturer.getName();
+        String manufacturerCountry = manufacturer.getCountry();
+        String insertFormatRequest = "INSERT manufacturers(name, country) VALUES (?, ?)";
+
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createManufacturerStatement =
+                        connection.prepareStatement(
+                             insertFormatRequest, Statement.RETURN_GENERATED_KEYS)) {
+            createManufacturerStatement.setString(1, manufacturerName);
+            createManufacturerStatement.setString(2, manufacturerCountry);
+            createManufacturerStatement.executeUpdate();
+            ResultSet generatedKeys = createManufacturerStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long id = generatedKeys.getObject(1, Long.class);
+                manufacturer.setId(id);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException(
+                    "Can`t insert manufacturer " + manufacturer.getName(), e);
+        }
```

Comment:
```suggestion
        } catch (SQLException e) {
            throw new DataProcessingException(
                    "Can`t insert manufacturer " + manufacturer, e);
        }
```

lets log all info about manufactrer